src/views/product-root-docs-path-landing/components/supported-icons.tsx

```diff
@@ -9,6 +10,7 @@ import { IconApi16 } from '@hashicorp/flight-icons/svg-react/api-16'
  * @TODO raise to a higher level in the project for reuse?
  */
 export const SUPPORTED_ICONS = {
+	docs: <IconDocs16 />,
```

Comment:
Added this icon, otherwise it was missing on non-`/docs` custom docs landing pages.